adapter/internal/oasparser/envoyconf/http_filters.go

```diff
@@ -158,6 +161,24 @@ func getLuaFilter() *hcmv3.HttpFilter {
 	return &luaFilter
 }
 
+// getHTTPLocalRateLimitFilter returns the local rate limit filter which is used for JWKS endpoint specifically.
+func getHTTPLocalRateLimitFilter() *hcmv3.HttpFilter {
+	localRateLimitConfig := &local_ratelimit_v3.LocalRateLimit{
+		StatPrefix: localRateLimitStatPrefix,
+	}
+	marshalledRateLimitConfig, err := ptypes.MarshalAny(localRateLimitConfig)
+	if err != nil {
+		logger.LoggerOasparser.Error("Error while generating the local rate limit filter.", err)
```

Comment:
Can we change this to `errorf` ?

Reply:
Errorf is only required if um going to format the error. But in this case, it is pretty much like right after the error message, we are printing the error